ui/app/appointments/views/manage/list/listView.html

```diff
@@ -49,34 +49,35 @@
     </div>
     <div ui-view="content@appointment"></div>
     <fieldset class="appointment-action-btn-container" ng-disabled="!selectedAppointment" ng-class="{'disabled': !selectedAppointment}">
-        <button show-if-privilege="{{manageAppointmentPrivilege}}" ng-click="editAppointment()">
+        <button show-if-privilege="{{manageAppointmentPrivilege}},{{ownAppointmentPrivilege}}" ng-click="editAppointment()"
+            ng-disabled="!isUserAllowedToPerform()">
             {{::'EDIT_APPOINTMENT' |translate}}
         </button>
-        <button show-if-privilege="{{manageAppointmentPrivilege}}"
+        <button show-if-privilege="{{manageAppointmentPrivilege}},{{ownAppointmentPrivilege}}"
```

Comment:
@sai789, add _\<space\>_ between `{{manageAppointmentPrivilege}}` and `{{ownAppointmentPrivilege}}`